test/ngResource/resourceSpec.js

```diff
@@ -750,6 +750,7 @@ describe('basic usage', function() {
     var json = JSON.parse(angular.toJson(cc));
     expect(json.$promise).not.toBeDefined();
     expect(json.$resolved).not.toBeDefined();
+    expect(json.$cancelRequest).not.toBeDefined();
```

Comment:
I just realized that this test uses `JSON.parse(angular.toJson(cc))`, which means it would pass even without this PR.
Can you add a new test (similar to this one) that uses `cc.toJSON()` directly? (Leave this one as is - it doesn't hurt.)